mdgen/src/main/java/uk/gov/ida/mdgen/MetadataGenerator.java

```diff
@@ -207,7 +207,7 @@ private EntityDescriptor buildEntityDescriptor() throws Exception {
         String xml = renderTemplate(nodeType.toString() + "_template.xml.mustache", yamlMap);
         EntityDescriptor entityDescriptor = ObjectUtils.unmarshall(new ByteArrayInputStream(xml.getBytes()), EntityDescriptor.class);
         entityDescriptor.setID(UUID.randomUUID().toString());
-        entityDescriptor.setValidUntil(DateTime.now().plusDays(validityDays));
+        entityDescriptor.setValidUntil(new DateTime(validityTimestamp));
```

Comment:
We're in a weird situation. Some of the changes to the metadata come from altering the entityDescriptor, and some come from filling in details through Mustache. I feel like we should use one or the other.